woocommerce/woocommerce-gutenberg-products-block
# Add new SearchListItem component

Extract the custom rendering from product categories, for re-use in product attributes. This lays some groundwork for #240 – rather than having to set up a long "renderCallback" with separate styles, we can use a single component, and share this between ProductCategoryControl & the upcoming ProductAttributeControl.

### How to test the changes in this Pull Request:

1. Check the category search in Products By Category, Best Sellers, On Sale, or Top Rated
2. Expect: there should be no visual or functionality change.

The markup of the items have changed slightly, which is why the test snapshot needed to be regenerated.
